recipes/jom/all/conanfile.py

```diff
@@ -0,0 +1,23 @@
+from conans import ConanFile, tools
+import os
+
+
+class JomInstallerConan(ConanFile):
+    name = "jom"
+    description = "jom is a clone of nmake to support the execution of multiple independent commands in parallel"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://wiki.qt.io/Jom"
+    license = "GPL-3.0"
+
+    settings = {"os" : ["Windows"]}
```

Comment:
I have some doubts about restricting settings this way, but it seems it worked just fine so... 🤔